test/internalisation/test-totalSupply-definition.kore.report

```diff
@@ -1980,7 +1980,7 @@ Symbols: 1720
   - inj
   - kseq
 Axioms: 352
-Axioms preserving definedness: 295
+Axioms preserving definedness: 296
```

Comment:
This change is caused by eliminating a redundant `.Set _Set_ OTHERSET` in a rule (by way of internalisation).

The rule has a `<activeAccounts> SetItem(ACCT) => .Set ... </activeAccounts>` (this field does not exist any more in the current evm-semantics config, btw).
In kore, the `...` becomes a variable `VAR`, and the  `.Set ...` on the RHS becomes `_Set_(.Set, VAR)`. Internalisation recognises the redundancy and removes the `_Set_` call; the rule is now marked as  preserving definedness.